corehq/apps/reports/static/reports/js/saved_reports.js

```diff
@@ -37,13 +37,13 @@ var ReportConfig = function (data) {
     self.unwrap = function () {
         var data = ko.mapping.toJS(self);
         var standardHQReport = hqImport("reports/js/standard_hq_report").getStandardHQReport();
-        if (null !== standardHQReport.slug) {
+        if (standardHQReport.slug) {
```

Comment:
Not sure if this was done intentionally but to allow `""` and restrict just `null` value. If you have checked that its a safe change then cool.